kai/routes/ws/get_incident_solution.py

```diff
@@ -0,0 +1,62 @@
+import json
+from typing import Iterator
+
+from aiohttp import web
+from aiohttp.web_request import Request
+from langchain_core.messages import BaseMessageChunk
+
+from kai.routes.util import to_route
+from kai.service.kai_application.kai_application import KaiApplication
+
+# TODO(@JonahSussman): Figure out proper pydantic model validation for this
```

Comment:
also add a TODO about figuring out if we should remove this and the other get_incident_solution